noxfile.py

```diff
@@ -46,9 +59,13 @@ def ci_checks_max(session: nox.Session) -> None:
     This is faster than running the checks separately, so it is suitable for CI.
 
     This does NOT run pytest_min, so that needs to be run separately as well.
+
+    Args:
+        session: the nox session.
     """
-    session.install(".[docs]", *PYTEST_DEPS, *FMT_DEPS, *LINT_DEPS,
-            *DOCSTRING_DEPS)
+    session.install(
+        ".[docs]", "mypy", "pylint", "nox", *PYTEST_DEPS, *FMT_DEPS, *DOCSTRING_DEPS
```

Comment:
why do we need `nox` inside a `nox` session?

Reply:
Yes. It installs a clean venv without `nox`.

Reply:
my question was "why?" :-)